### DISCUSSION TITLE
Vaultwarden image with PostgreSQL compatibility?

### DISCUSSION THREAD

**Discussion by mzch** [dani-garcia/vaultwarden#1643] [Q&A] [answered]
There aren't images `vaultwarden/server_postgresql` on Docker Hub.
Do you create a `vaultwarden` image with PostgreSQL compatibility?

**Comment by BlackDex** [comment, marked as answer]
The default image had Postgresql support together with sqlite and MySQL all in one image.

Also read: https://github.com/dani-garcia/vaultwarden/releases/tag/1.21.0

**Reply by mzch** [reply]
Oh, I missed it. Thanks! :)
